src/store/getters/state-getters.ts

```diff
@@ -454,3 +455,12 @@ export const getStaffPaymentStep = (state: StateIF): StaffPaymentStepIF => {
 export const getCourtOrderStep = (state: StateIF): CourtOrderStepIF => {
   return state.stateModel.courtOrderStep
 }
+
+/** The custodian email. */
+export const getCustodianEmail = (state: StateIF): string => {
+  return 'hardcoded.custodian@email.com' // TODO: Change this once custodian details are available in state
```

Comment:
Done now?

Reply:
I'l make this connection in my current work 👍